docs/development_suite/project/overview.md

```diff
@@ -0,0 +1,191 @@
+
+From here you are able to access individual pod logs. If necessary, you can manually select and restart pods.
+
+[More information about Runtime here](/development_suite/monitoring/introduction.md)
+
+### Debug
+
+The Debug section shows the list of all the microservices developed with the Developer Console.
+At the first level you have the necessary information to understand the health of the service: whether it is active or not, and the memory levels used.
+
+The real potential of the debug area is its connection with telepresence. Pressing the "Debug" button will in fact display a string to be pasted on your terminal that simulates the behavior of your microservice in a real context. Telepresence is an open source tool that lets you run a single service locally, while connecting that service to a remote Kubernetes cluster.
+
+[More information about Debug here](/development_suite/debugging/telepresence.md)
+
+### Documentation
+
+The Documentation section is a vital part of any Restful API in order to drive consistency and discipline across your API development workflow.
+
+The Documentation section in the Developer Console area exposes the Open API documentation of all services and CRUDs that you have defined and exposed. In this way, you will be able to obtain information about the routes exposed by your resources from a single section, and to test the correct functioning of each of them. You can share API documentation both internally within your company, and externally to partners and suppliers.
+
+To test each API, you can access your tags on the right side of the screen, where the APIs are grouped according to their tags: by clicking on one tag, you will visualize all the APIs that belong to that tag.
+
+[More information about Documentation here](/development_suite/api-portal/api-documentations.md)
+
+### Marketplace
+
+The Marketplace section contains several code resources to develop your microservices. You can also access Mia-Platform Marketplace from the Design area of Console, when you are creating a new microservice.
+
+You will see a set of Plugins, Examples, Templates and Applications that you can choose from to easily set-up a single microservice or a bundle of microservices with predefined and tested functionalities.
+
+You can start from a:
+
+* Plugin: a ready-to-use Microservice powered by Mia-Platform or a Certified Partner;
+* Template: a base repository (open source in the Mia-Platform Marketplace GitHub page) from which you can start to create a new Microservice;
+* Example: a specific use-case, a ready-to-use model to create your Microservice;
+* Application: a bundle of resources including Plugins, Templates and Examples, or resources like CRUD, Endpoints, etc;
+* Docker Image Name: an existing Docker image of a Microservice.
+
+[More information about Marketplace here](/marketplace/overview_marketplace.md)
+
+### What's new
```

Comment:
This is already out in production. However, I think this paragraph has to be put outside project overview. It could be placed at console overview level. What do you think @akelity ?

Reply:
Yes I agree

Reply:
Also, in the Project Configuration page we read "Console sections" in many occasions, while I think it should be more appropriate to address them as Project sections